sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,45 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
+    """
+    Returns the boolean value of dummify to be used.
```

Comment:
This is confusing as written. Better would be "Returns whether \`\`symbol\`\` should be dummified"